include/gsm0710muxer/muxer_impl.h

```diff
@@ -137,7 +137,7 @@ inline int Muxer<StreamT, MutexT>::start(bool initiator, bool openZeroChannel) {
             CHECK_TRUE(channelEvents_, GSM0710_ERROR_NO_MEMORY);
         }
 
-        xEventGroupClearBits(events_, EVENT_MAX - 1);
+        xEventGroupClearBits(events_, ~((0x01UL << GSM0710_EVENT_BASE) - 1));
```

Comment:
`EVENT_STATE_CHANGED`?

Reply:
Because
```c
EVENT_STATE_CHANGED          = 0x01 << GSM0710_EVENT_BASE,
```

Reply:
The intent is to clear anything above `GSM0710_EVENT_BASE`, having `EVENT_STAGE_CHANGED` here confuses what we actually are trying to do.